discovery-infra/bootstrap_in_place.py

```diff
@@ -173,7 +165,7 @@ def log_collection(controller, vm_ip):
     try:
         logging.info("Gathering sosreport data from host...")
         node = Nodes(controller, private_ssh_key_path=SSH_KEY)[0]
-        gather_sosreport_data(node)
+        gather_sosreport_from_node(node)
     except Exception:
```

Comment:
no, guys, no
``KeyboardInterrupt`` will be caught as well when catching ``BaseException``
catching ``Exception`` is bad enough as is :)

Reply:
You are right if we want catch only python built-in exceptions, but in case of using an external libraries, how can you tell if some exceptions inherits directly from `BaseException`?
On large projects I believe it is better to use `BaseException` if we want to catch all and not trace who inherit from `Exception` and who from   `BaseException` 🤷🏻‍♂️